crates/ott-balancer/src/lib.rs

```diff
@@ -33,6 +33,23 @@ pub async fn run() -> anyhow::Result<()> {
     BalancerConfig::load(&args.config_path)?;
     let config = BalancerConfig::get();
 
+    let _validate = Arg::new("validate")
+        .short('v')
+        .long("validate")
+        .help("Validate the configuration and exit");
```

Comment:
Why is this not in `config::Cli`? This makes it so that the help text doesn't include the flag.

Reply:
I figured that out after looking at the docs again. I was just about to change that.